mkdocs/docs/how-to-release.md

```diff
@@ -130,6 +130,15 @@ Run the [`Python release` Github Action](https://github.com/apache/iceberg-pytho
 * Tag: Use the newly created tag.
 * Version: Set the `version` to `main`, as the source cannot be modified.
 
+Or if `gh` is available, trigger the action using:
+
+```bash
+gh workflow run python-release.yml --repo apache/iceberg-python --ref ${GIT_TAG} --field version=main
+SVN_RUN_ID=$(gh run list --repo apache/iceberg-python --json databaseId -q '.[0].databaseId')
```

Comment:
I think this takes the latest run, would that be safe? Or should we do some more filtering